### PR TITLE
add default keymap for kira80

### PR DESCRIPTION
**Pull request by elmo-space**

### CODE REVIEW

**Comment by noroadsleft** [comment]
I need you to remove the non-kira80 commits. One of them is bugged and prevents Configurator from deploying; we fixed it this morning but it required overwriting the existing history.

```shell
git checkout add_kira80_default_keymap
git fetch upstream master
git reset --hard FETCH_HEAD
git cherry-pick 9972438136d87215ec483e86f3e56407833d1a3e
git push --force-with-lease
```

File: public/keymaps/k/kira80_default.json [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+      "KC_ESC",             "KC_F1",   "KC_F2",   "KC_F3",   "KC_F4",   "KC_F5",   "KC_F6",   "KC_F7",   "KC_F8",   "KC_F9",   "KC_F10",  "KC_F11",  "KC_F12",                  "KC_PSCR", "KC_SLCK", "KC_PAUS",
+      "KC_GRV",  "KC_1",    "KC_2",    "KC_3",    "KC_4",    "KC_5",    "KC_6",    "KC_7",    "KC_8",    "KC_9",    "KC_0",    "KC_MINS", "KC_EQL",  "KC_BSPC", "KC_BSPC",      "KC_INS",  "KC_HOME", "KC_PGUP",
+      "KC_TAB",  "KC_Q",    "KC_W",    "KC_E",    "KC_R",    "KC_T",    "KC_Y",    "KC_U",    "KC_I",    "KC_O",    "KC_P",    "KC_LBRC", "KC_RBRC", "KC_BSLS",                 "KC_DEL",  "KC_END",  "KC_PGDN",
+      "KC_CAPS", "KC_A",    "KC_S",    "KC_D",    "KC_F",    "KC_G",    "KC_H",    "KC_J",    "KC_K",    "KC_L",    "KC_SCLN", "KC_QUOT",            "KC_ENT",
+      "KC_LSFT", "KC_NUBS", "KC_Z",    "KC_X",    "KC_C",    "KC_V",    "KC_B",    "KC_N",    "KC_M",    "KC_COMM", "KC_DOT",  "KC_SLSH", "KC_RSFT", "KC_TRNS",                            "KC_UP",
+      "KC_LCTL", "KC_LGUI", "KC_LALT",                                  "KC_SPC",                                   "KC_RALT", "KC_RGUI", "MO(1)",   "KC_RCTL",                 "KC_LEFT", "KC_DOWN", "KC_RGHT"
```
Comment (noroadsleft) [inline public/keymaps/k/kira80_default.json]: You have RGUI and MO(1) in the opposite order on qmk_firmware.

```suggestion
      "KC_LCTL", "KC_LGUI", "KC_LALT",                                  "KC_SPC",                                   "KC_RALT", "MO(1)",   "KC_RGUI", "KC_RCTL",                 "KC_LEFT", "KC_DOWN", "KC_RGHT"
```

Comment (elmo-space) [reply public/keymaps/k/kira80_default.json]: oh ok that's just how I usually do it 😄

**Comment by noroadsleft** [comment]
Thanks!

**Pull request merged**